src/app/api/internal-users/route.ts

```diff
@@ -0,0 +1,21 @@
+import { errorHandler } from '@/utils/common';
+import { CopilotAPI } from '@/utils/copilotApiUtils';
+import { NextResponse, NextRequest } from 'next/server';
+import { z } from 'zod';
+
+export async function GET(request: NextRequest) {
+  const searchParams = request.nextUrl.searchParams;
+  const token = searchParams.get('token');
+  if (!token) {
+    return errorHandler('Missing token', 422);
+  }
+
+  const copilotClient = new CopilotAPI(z.string().parse(token));
+  try {
+    const clients = await copilotClient.getInternalUsers();
+
+    return NextResponse.json(clients);
+  } catch (error) {
+    return errorHandler('Clients not found.', 404);
```

Comment:
lets add a log of the error here.. so we can debug later

Reply:
70ec9ca1f192af7df4413cb663eb9c625b139cea commit resolved this review